rebar.config

```diff
@@ -107,7 +107,17 @@
 
         % for hyperchains and the testing suite
         {aesophia_aci_encoder, {git, "https://github.com/aeternity/aesophia_aci_encoder",
-          {ref, "110813d"}}}
+          {ref, "110813d"}}},
+
+        % all four for devmode:
+        {ebip39,           
+                {git, "https://github.com/aeternity/ebip39.git", {ref, "7520340"}}},
+        {eaex10,
+                {git, "https://github.com/aeternity/eaex10.git", {ref, "958046a"}}},
+        {epbkdf2, 
+                {git, "https://github.com/aeternity/epbkdf2", {tag, "v1.0.0"}}},
+        {ec_utils,
+                {git, "https://github.com/hanssv/ec_utils.git", {ref, "e557b1c"}}}
```

Comment:
Nope, the node should not depend on these 🙀

Reply:
I'm not opinionated about where to put them, maybe @uwiger can hint me how to make this available in his `setup` [hook function](https://github.com/nikita-fuchs/aeplugin_dev_mode/blob/e9d921b9b51a5b895486b5744fde9bb9de98ba9d/src/aeplugin_dev_mode.app.src#L13)  ? I tried grasping the workings of `setup` et al., but to my best understanding only the _node's_ dependencies are available at this point. Open to find out what I missed and learn a thing or two.

For curiosity, what's so bad about having those in the node deps after all @hanssv ?